### DISCUSSION TITLE
Already add CRUD Menu, and set Permission but it won't shown in the roles

### DISCUSSION THREAD

**Discussion by buncizs** [uasoft-indonesia/badaso#341] [Q&A] [answered]
Hi, Thank you for making this awesome Builder, i found it while scrolling in the social media and surely looks so promising
so i decided to try and i just thought if it worth, maybe i'll recommend this to my compatriot. Surely, after new fresh install with the pre requisite and your official walkthrough which is very clear and straight forward. i met some problem, with this project setup
here are the scenario
- i make another tables to try the CRUD, and make it listed on the CRUD Menu too
- after that, i make some user and roles lets say Roles=Operator and new user Jhon which the permission i set only can see the data on that table and just add into that table
- while it shown to the user John the menu new table, Jhon user's could also see what is inside the table, so Read wasn't the problem here,
- when i try to add, there's none shown in John user, but in administrator it show full list
- i noticed too when i click on add in the little blue rectangle in right side, there are some popup DANGER in red block rectangle in bottom right side but as i click it, nothing happens.

i know you were busy creating great software and a question by some random people that try your software less matter but i would be glad if you answer my question. 

here are my image when its not show
![image](https://user-images.githubusercontent.com/7616125/131287190-86b98669-57f4-4575-8488-9a2cc777c655.png)

here are my image when its fully shown
![image](https://user-images.githubusercontent.com/7616125/131287277-5c47ae1d-f073-4a36-81fe-58fb54fa5d67.png)

**Comment by buncizs** [comment, marked as answer]
after examining a little more further with help from browser console, i found that you need permission on read crud data and browse crud data to fix the problem
so would be nice if it can be untied separately because i'm afraid this little thing will exploited because it opens for attacker to capture what the data streamed through

**Reply by rizkiheryandi** [reply]
Thank you for using badaso

yes we also found this issue

we will fix it in the next release

Thank you in advance
